SQL/beestation_schema.sql

```diff
@@ -496,7 +496,7 @@ CREATE TABLE IF NOT EXISTS `SS13_schema_revision` (
   `date` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
   PRIMARY KEY (`major`,`minor`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
-INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (7, 0);
+INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (7, 1);
```

Comment:
need to update `DB_MINOR_VERSION` in `code/__DEFINES/subsystems.dm`

It is infuriating how many places this is in lol. that's the last one